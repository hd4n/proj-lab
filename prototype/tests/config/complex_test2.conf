#31. Komplex teszteset 2
#@author Eros Pal
#@since 2022-04-23

@items
nu n1
am a1

@effects
da d1 5
im i1 5

@agents
vi ag1 d1

@codes
co c1 ag1 1 1
co c2

@fields
em e1
em e2
la l2 c2
em e4
la l1 c1

@neighbors
e1 l1
l1 e2
e2 l2
e4 l1

@virologists
v1 e1 null a1;n1
v2 l2 null null i1