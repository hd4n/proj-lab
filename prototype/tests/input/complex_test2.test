load complex_test2.conf
step v1 l1
nextround
interact v1
interact v2
step v2 e2
craft v1 c1
nextround
useagent v1 v2 ag1
nextround
step v1 e4
nextround
step v2 l1
nextround
interact v2
getstats v1
getstats v2