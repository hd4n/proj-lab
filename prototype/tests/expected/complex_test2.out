Load: sikeres
step: v1 l1
nextround: sikeres
interact: v1 l1
interact: v2 l2
step: v2 e2
craft: v1 c1
nextround: sikeres
useagent: v1 v2 ag1
nextround: sikeres
step: v1 e4
nextround: sikeres
step: v2 l1
nextround: sikeres
interact: v2 l1

stats: v1
	fi_e4
	di_e4
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_0
	neededNucleo_0
	co_c1
	ma_null
	ag_null
	eq_null
	eqa_null
	ef_null

stats: v2
	fi_l1
	di_l1
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_c2+c1
	ma_null
	ag_null
	eq_null
	eqa_null
	ef_null
